app/controllers/copies_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CopiesController < ApplicationController
+  before_action :get_book
+  before_action :set_copy, only: [:show, :edit, :update, :destroy]
+  def index
+    @copies = @book.copies
+  end
+
+  def new
+    render "./books/copies/new"
+    @copy = @book.copies.build
```

Comment:
Line 10 will need to be before the render if you want `@copy` to be available in the view.